app/views/govuk_publishing_components/components/_input.html.erb

```diff
@@ -119,6 +121,18 @@
     <%= tag.div class: "govuk-input__wrapper" do %>
       <%= input_tag %><%= tag.span suffix, class: "govuk-input__suffix", aria: { hidden: true } %>
     <% end %>
+  <% elsif reveal_password %>
+    <%= tag.div class: "gem-c-input--reveal-password",
```

Comment:
Do we not want the wrapper here? it feels a bit surprising to not have it.

Reply:
Oh the `govuk-input__wrapper`? That's a good point. Odd. Seems to render fine without it, but I'll take a look, thanks.

Reply:
Have investigated. It's basically unnecessary - `govuk-input__wrapper` adds some flexbox styles, which sort of help with this layout but don't really.

Reply:
Sure, thanks for looking into it. If it's unnecessary, but not detrimental, I wonder if it's better to include it as otherwise this is just a special case for this modifier which makes it a bit more special.

One thing that's a bit weird here is that we have this `gem-c-input--reveal-password` class on this div, yet we have `gem-c-input` set on the input (the input seems the wrong place really 🤔 ). I wonder, to match the conventions here, whether this should be `gem-c-input__wrapper--reveal-password`?

Reply:
Ah yes, probably, good shout. Have changed.